src/main/views/account-home.njk

```diff
@@ -0,0 +1,28 @@
+{% extends "template.njk" %}
+{% from "./macros/common-components.njk" import goBack %}
+
+{% block pageTitle %}
+    {{ title }}
+{% endblock %}
+{% block beforeContent %}
+    {{ super() }}
+    {{ goBack() }}
+{% endblock %}
+{% block content %}
+    <div class="govuk-grid-row">
+        <h1 class="govuk-heading-l">{{ heading }}</h1>
+    </div>
+    <div class="cards-wrapper">
+        {% for card in cards %}
+            <div class="govuk-!-margin-top-2 govuk-!-margin-bottom-2">
+                <div class="account-card govuk-!-margin-left-2 govuk-!-margin-right-2">
+                    <p>
+                        <a href="{{ card.link }}"
+                           class="govuk-link govuk-body govuk-!-font-weight-bold">{{ card.title }}</a>
+                    </p>
+                    <p class="govuk-body">{{ card.description }}</p>
+                </div>
+            </div>
+        {% endfor %}

```

Comment:
this is nice and neat!

Reply:
Thanks,
resolving as it is blocking the merge 😂